src/ziggurat/util/java_util.clj

```diff
@@ -0,0 +1,59 @@
+(ns ziggurat.util.java-util
+  (:require [clojure.string :as str])
+  (:import (java.util Map Arrays)))
+
+(declare create-clojure-hash-map)
+(declare create-clojure-vector)
+(declare create-clojure-vector-from-array)
+
+(defn- is-java-array?
+  [array]
+  (str/starts-with? (.getName (.getClass array)) "["))
```

Comment:
is there a better way to check if an object is a java array?

Reply:
@theanirudhvyas Clojure does not allow us to use String[].class. Also, names for Array classes begin with a "[". It's explained in Oracle JVM doc at [1].

[1] https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#jvms-4.3.2-200